warehouse/query-core/src/test/java/datawave/query/FilterFieldsQueryTest.java

```diff
@@ -196,6 +196,28 @@ public void testEqCityAndEqContinentBlackList() throws Exception {
         }
     }
     
+    @Test
+    public void testRegexBlacklist() throws Exception {
+        log.info("------  testRegexBlacklist  ------");
+        
+        String regex = "miss.*";
+        for (final TestCities city : TestCities.values()) {
+            String query = CityField.CITY.name() + " == '" + city.name() + "' and " + CityField.STATE.name() + " =~ '" + regex + "'";
```

Comment:
Minor Nit: Refactor to use RawDataManager.*_OP constants for consistency(?)

Reply:
Originally, " == " was used and was later converted to use the constant. Must have missed this one.